Assets/Scripts/Weapon/ShootWeapon.cs

```diff
@@ -49,17 +49,17 @@ private void handleKill(RaycastHit hit)
     {
         if (hit.transform.tag == "CorrectAnswer")
```

Comment:
Wait, you're using string comparison to see if the answer matches?
Is there no other possible approach?
In the worst case, the least you could do is store the string literals as constants.

Reply:
Feared so.
Then at least compare constants, and not string literals.

Reply:
done